src/components/Button/__tests__/Button-test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Button from '../Button';
+
+it('renders correctly', () => {
+    const tree = renderer.create(
+            <Button>Hello</Button>
+              ).toJSON();
+      expect(tree).toMatchSnapshot();
+});
+
+
```

Comment:
expand test to include all props combos. This way we can add a snapshot serializer to render out the classes and html which allows us to see any style diffs when the theme changes